tests/subsys/storage/flash_map/overlay-mbedtls.conf

```diff
@@ -1 +1,2 @@
 CONFIG_FLASH_AREA_CHECK_INTEGRITY_MBEDTLS=y
+CONFIG_ZTEST_NEW_API=y
```

Comment:
don't need to add it into this file, it is an overlay based on prj*.conf, so you just need to modify prj*.conf